pages/seed/index.mdx

```diff
@@ -0,0 +1,59 @@
+# What is Seed?
+
+**Seed** is a set of tools **understanding your database schema** to help you **generate production-accurate data**.
+
+It consists of the following parts:
+
+- **Seed ORM**: Auto-generated and type-safe data client for Node.js & TypeScript 🛡️
+- **Seed CLI**: A CLI to generate and keep the data client in sync with your database 🛠️
+- **Seed AI**: Our custom model identifying the shape of your data 🤖
+
+We are compatible with [PostgreSQL]() and [SQLite](). MySQL support is coming soon.
+
+## Seed ORM
+
+Our data client is auto-generated from your database schema.
+We are using the full power of TypeScript to provide you with a type-safe and auto-completed experience when working with your data.
+
+You can focus on the data that matter to **you**, we will take care of generating all the relationships. ✨
+
+Here is a sneak peek of what you can do with Seed ORM:
+
+Are you working on a blog app and need 5 posts?
+```ts
+await seed.posts((x) => x(5))
+```
+
+Or maybe you need a user with a specific email?
+```ts
+await seed.users([
+  { email: 'snappy@snaplet.dev' },
+])
+```
+
+What if you want to create an organization with a varying number of members?
+```ts
+await seed.organizations([
+  {
+    name: 'Snaplet',
+    members: (x) => x({ min: 1, max: 10 }),
+  },
+])
+```
+
+And this is just the beginning! Explore the [Seed ORM guide]() to see all the possibilities.
+
+## Seed CLI
+
+The CLI is the entry point to Seed. It will help you generate the data client and keep it in sync with your database.
+
+To begin your journey with Seed in seconds, run this command in your project directory:
+```bash >_&nbsp;terminal
+npx @snaplet/seed init
+```
+
+Checkout the [Seed CLI reference]() to learn more about the CLI and how to use it.
+
+## Seed AI
+
+We are using AI to have a deeper understand of your datatabase schema so we can generate the most accurate and production-like data for you.
```

Comment:
This content looks like a duplicate of `overview.mdx` - which one should stay?

Reply:
I'll delete this one!